### ISSUE TITLE
Cannot connect terminal to remote host in remote vscode session

### ISSUE THREAD

**Issue by jordangarrison** [microsoft/vscode-remote-release#7039]
Type: <b>Bug</b>

Remote session is working but when I open the terminal it is blank and won't take input.

Extension version: 0.84.0
VS Code version: Code 1.70.0 (da76f93349a72022ca4670c1b84860304616aaa2, 2022-08-04T04:39:20.716Z)
OS version: Darwin arm64 21.5.0
Modes:
Remote OS version: Linux x64 5.15.56

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 Pro (10 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|5, 4, 3|
|Memory (System)|32.00GB (3.08GB free)|
|Process Argv|--crash-reporter-id dee7574e-ef1a-4ebe-9244-132fbaf73601|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: endeavour|
|OS|Linux x64 5.15.56|
|CPUs|AMD Ryzen 9 3900X 12-Core Processor (24 x 3800)|
|Memory (System)|62.74GB (49.85GB free)|
|VM|0%|
</details>

**Comment by jinzcdev** [comment]
What is the terminal you use? bash or zsh?

**Comment by jordangarrison** [comment]
I found the issue. The shell for vscode writes copies of your rc files for the shell you use in the `/tmp/vscode-zsh` directory on the host machine. The `/tmp/vscode-zsh` folder became permission locked after upgrading from vscode 1.69 to 1.70 on the linux host (NixOs). This prevented both the remote and local host from creating new zsh sessions. Once I removed the `/tmp/vscode-zsh` directory I was able to start the shell when vscode recreated the folder and populated the rc files correctly.
